config/runtime/base/lava.jinja2

```diff
@@ -19,16 +17,16 @@
 
 job_name: "[KernelCI AKS] {{ node.id }} {{ node.name }} {{ node.revision.describe }}"
 
-device_type: qemu
+device_type: {{ platform_config.base_name|default(platform_config.name) }}
```

Comment:
The base name is the only valid one to use here.  The `name` attribute is only something in YAML config to have several variants.

Note: I wonder if it might actually be better to ultimately decouple the templates from the config object attributes, so just having `{{ platform_type }}` would work better, and each Runtime implementation can populate accordingly.  But that's a bit beyond the scope of this PR.

Reply:
@nuclearcat This hasn't been addressed, the PR was merged too early.

Reply:
Tracked by #2168, fixed in #2169